ckan/lib/search/jobs.py

```diff
@@ -0,0 +1,70 @@
+# encoding: utf-8
+
+# (canada fork only): background search index rebuilding
+#TODO: upstream contrib!!
+
+import json
+import datetime
+from rq import get_current_job
+
+import ckan.logic as logic
+import ckan.lib.search as search
+import ckan.model as model
+
+from ckan.plugins import toolkit
+
+from logging import getLogger
+log = getLogger(__name__)
+
+
+def reindex(package_ids=None, group_id=None):
```

Comment:
add docstring to describe parameters and behaviour?

Reply:
also we're going to be indexing things other than packages in the future, so how about calling this function `reindex_packages`?